index.html

```diff
@@ -42,6 +42,19 @@
             <p class="branding-info">Senior project by <a href="https://erikboesen.com">Erik Boesen</a> '19</p>
             <p class="branding-info">Maintained by GMHS <a target="_blank" rel="noopener" href="https://apc-gm.com">A.P.C.</a></p>
         </div>
+        <div id="statistics">
+            <img id="statistics-button" src="/img/chevron_left_white_24dp.svg">
+            <h3 id="statistics-header">
+                Class of <span id="statistics-header-class" class="data"></span> stats
+            </h3>
+            <ul id="statistics-stats">
+                <li><span id="stats-students" class="data"></span> students displayed</li>
```

Comment:
Nit: `data` is a fairly vague name. Perhaps choose something more specific (though still succinct)?